graphql_compiler/cost_estimation/statistics.py

```diff
@@ -97,16 +115,39 @@ def get_field_quantiles(self, vertex_name, field_name):
         """
         return None
 
+    def get_value_count(self, vertex_name: str, field_name: str, value: Any) -> Optional[int]:
```

Comment:
Is `int` the right type hint here? Since we're estimating, do we always want integer-typed estimates? Is `1.47` not more useful than `1`?

Reply:
It is, but the smallest numbers we'll practically return are around `50`, and `51/50` is very close to `1`. I have no strong feelings either way, and I can use `float` if that's simpler to you.

Reply:
I'd say that all other statistics are float-typed because we generally deal in estimates, so let's make this one `float` too if it's all the same to you.